### PR TITLE
Fix circular imports

### PR DESCRIPTION
**Pull request by rfratila**
I apologize for the overlap with the freeze weights PR (not exactly sure why it did that but anyway). I just moved the get_notable_indices to the utils for plotters. you can now have it like we did before with `from vulcanai2.models import DenseNet` instead of `from vulcanai2.models.dnn import DenseNet`

### CODE REVIEW

**Comment by rfratila** [comment]
Actually would it make more sense to set `requires_grad= ~freeze`? we wouldn't need all the conditional statements but it's harder to read. thoughts @joemehltretter @nirtiac @priyatharsan ?

File: vulcanai2/plotters/utils.py [hunk 1]
```diff
@@ -38,10 +63,10 @@ def __init__(self, network):
     def _hook_top_layers(self):
         def hook_function(module, grad_in, grad_out):
             # TODO: Revisit dim disorder and check isinstance for classes.
-            if module.__class__.__name__ == 'Linear':
+            if isinstance(module, torch.nn.Linear):
                 # grad_in shape is (bias, input, weights)
                 self.gradients = grad_in[1]
-            elif module.__class__.__bases__[0].__name__ == '_ConvNd':
+            elif isinstance(module, torch.nn.modules.conv._ConvNd):
```
Comment (priyatharsan) [inline vulcanai2/plotters/utils.py]: Just curious, is there any diff wrt to performance by doing this? instead of checking with base Class name?

Comment (nirtiac) [reply vulcanai2/plotters/utils.py]: my impression was because it lets you deal with complex class hierarchies

Comment (priyatharsan) [reply vulcanai2/plotters/utils.py]: I don't get it. What do you mean?

Comment (priyatharsan) [reply vulcanai2/plotters/utils.py]: 👍  thanks

**Pull request merged**